src/entities/Peoples.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import { gql } from "apollo-boost";
+import { Query } from "react-apollo";
+import { useParams } from "react-router";
+
+
+import Paginator from '../Paginator.js'
+
+import './customcards.css'
+
+import { Button, Container, Table} from 'react-bulma-components';
+
+const Q = gql`
```

Comment:
Currently you have all the functionalities in just one component. This file is calling jquery and also responsible for rendering it, which is a bad practice. Try to divide this component as per the concern.

Reply:
Got it, ill rewrite the code